### ISSUE TITLE
TimePicker tab navigation

### ISSUE THREAD

**Issue by jamesport079** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3650]
### Bug explanation

If TimePicker control has keyboard focus, pressing Shift-Tab (move to previous tab stop) does not work. Pressing Tab (move to next Tab stop) works normally. 

### Version

5

**Comment by corvinsz** [comment]
created a [PR #3652](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/3652) that should fix this issue
